lib/ansible/modules/web_infrastructure/ansible_tower/tower_project.py

```diff
@@ -104,7 +104,7 @@ def main():
         scm_type=dict(choices=['manual', 'git', 'hg', 'svn'], default='manual'),
         scm_url=dict(),
         scm_branch=dict(),
-        scm_credential=dict(),
+        scm_credential=dict(default=None),
```

Comment:
Isn't `None` the default value [already](https://github.com/ansible/ansible/blob/3bb9f689e954f86e1245c97cf16914b5316de5b5/lib/ansible/module_utils/basic.py#L2123) ?

Reply:
Hmm, good point, I somehow felt the need to make it explicity. Let me fix that.